### DISCUSSION TITLE
Project seems abandoned, latest release has many bugs.

### DISCUSSION THREAD

**Discussion by souporman** [rg35xx-cfw/Koriki#117] [General]
You can't use Dreamcast, the ROMs partition is still in EXT4, the battery only charges under specific circumstances while using this build... and that's just what I've seen.  Only use this if you have a 2nd SD-card and you feel like goofing around.

**Comment by italianbeef12** [comment]
I would look forward to an update, but I don't agree with your assessment. Dreamcast is an analog stick console, so the 35XX isn't the right platform to play Dreamcast games. The EXT4 partition is a one time fix and easy to work around with a partition manager. And I don't have problems with charging and sleeping.

The flip side is that this is a *beautiful* front end and I really enjoy using it. Much nicer than any of the other front ends I've tried.

**Comment by souporman** [comment]
Dreamcast works on all of the other OSes for the RG35XX that I've tried, and there are many games that play fine.  Fixing bugs and ignoring bugs doesn't make them any less bugs.  Anyway, it's moot now, as I've upgraded to the H, and given the XX away.
